src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationConfiguration.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Contains well-known configuration settings.
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Required for serialization.")]
+    public class SuperOfficeAuthenticationConfiguration
```

Comment:
What's this class actually for? I can't see any use of it anywhere by the provider code, other than being used by `SuperOfficeAuthenticationConfigurationManager`, which itself isn't used by anything.